src/viewer.js

```diff
@@ -31,9 +31,13 @@ function runVizViewer(pythonPath, reportPath, port)
             vscode.window.showErrorMessage("Please install vizviewer to view the report");
         } else if (m = regex.exec(message)) {
             let url = m[1];
-            openInWebview(url, path.basename(reportPath), () => {
-                process.kill();
-            });
+            vscode.env.asExternalUri(vscode.Uri.parse(url)).then(
+                externalUri => {
+                    openInWebview(externalUri, path.basename(reportPath), () => {
```

Comment:
This is a great patch. There's a small thing here: `openInWebview` originally expects a string `url` which is used in the template html. This works now because `Uri` object implements its own `toString()` method. But I think it would be easier to follow if we make it explicit here - to convert the `Uri` object `externalUri` here to a string. So maybe pass `externalUri.toString()`?